modules/webrtc_aec/aec.cpp

```diff
@@ -106,6 +108,13 @@ int webrtc_aec_alloc(struct aec **stp, void **ctx, struct aufilt_prm *prm)
 
 	aec->inst->gain_control()->Enable(true);
 
+	if (webrtc_aec_extended_filter) {
+		config.Set<webrtc::ExtendedFilter>(
+			new webrtc::ExtendedFilter(true)
+		);
+	}
```

Comment:
please add an empty line

Reply:
Before line 112? 

Sorry, i didn't understand currently the coding style.

Reply:
After a `}` normally we have an empty line.

Reply:
Fixed with c3c6e88b